tests/onboarding/SignupTest.spec.ts

```diff
@@ -0,0 +1,39 @@
+import {test, expect} from '@playwright/test'
+import SignupPage from '../../page-object/onboarding/SignupPage'
+import Homepage from '../../page-object/home/HomePage'
+import UtilityHelper from '../../utility/UtilityHelper'
+import DeletePage from '../../page-object/onboarding/DeletePage'
+const testData = JSON.parse(JSON.stringify(require('../../test-data/test-data.json')))
+
+
+test('Signup',async({page})=> {
```

Comment:
Description should be `Register a new user` and add space before =>

Reply:
fixed